### ISSUE TITLE
Distances in Report

### ISSUE THREAD

**Issue by michaelwittmann** [amodeus-science/amodeus#367]
Hey everyone, first of all thanks for the great work on amodeus so far. 
I played around a bit to verify if amodeus fits to my needs for simulating taxi services in Munich. 

Actually I'm struggeling with the values given in the Simulation report. It seams, that the distances calculated do not match to the real driven distances inside the simulation. Is there any know Issue related to this. 

To get closer I tried a simple run with one vehicle and two requests. The two rides have an original distance of 7,8 km

The report shows that the total distance with customer for example is 4,30 km, where the total distance taken from the `output_plans.xml` is 7,2 km. Due to other routes driven in the simulation this should be the correct distance.  

![image](https://user-images.githubusercontent.com/25772536/69875940-96b8c880-12be-11ea-8d78-a6df10b371a4.png)

```XML
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE population SYSTEM "http://www.matsim.org/files/dtd/population_v6.dtd">

<population>

	<attributes>
		<attribute name="coordinateReferenceSystem" class="java.lang.String" >EPSG:25832</attribute>
	</attributes>


<!-- ====================================================================== -->

	<person id="74106653">
		<plan score="16.371905150264233" selected="yes">
			<activity type="activity" link="81488" x="689102.4713871534" y="5336865.173642203" end_time="00:00:11" >
			</activity>
			<leg mode="av" dep_time="00:00:11">
				<route type="av" start_link="106689" end_link="10888" trav_time="00:00:00" distance="5730.515483306246">{"inVehicleTime":"NaN","price":"NaN","waitingTime":300.0,"operatorId":"op1"}</route>
			</leg>
			<activity type="activity" link="10888" x="693285.7122613986" y="5334743.275227725" >
			</activity>
		</plan>

	</person>

<!-- ====================================================================== -->

	<person id="74106660">
		<plan score="16.682785827886924" selected="yes">
			<activity type="activity" link="26840" x="691752.5100729803" y="5335422.55546116" end_time="00:00:44" >
			</activity>
			<leg mode="av" dep_time="00:00:44">
				<route type="av" start_link="109926" end_link="41942" trav_time="00:00:00" distance="1511.421616468973">{"inVehicleTime":"NaN","price":"NaN","waitingTime":300.0,"operatorId":"op1"}</route>
			</leg>
			<activity type="activity" link="41942" x="690526.904471905" y="5334972.770260792" >
			</activity>
		</plan>

	</person>
</population>
```

**Comment by claudioglasses** [comment]
Hi Michael, would you be so kind and let me know which version you use? Please pull master and try with the newest version.  In old versions, the processing was faster but full accurarcy was only possible if the export time constant was set to 1 second. Default was 10 s. 
Additionally, there was one issue a while ago which was related to the sampling time. Then, you have to be careful what unit the coordinated system that you use is in... not all of the EPSG systems are in meter.

**Comment by michaelwittmann** [comment]
Thanks @claudioglasses.
I use version `1.8.9` as given in the `pom.xml` from the amod repository

```XML
<dependency>
    <groupId>ch.ethz.idsc</groupId>
    <artifactId>amodeus</artifactId>
    <version>1.8.9</version>
</dependency>
```




You mentioned setting the time constant, is this possible in the config file? 

It really seems to be an post-processing issue. As I said, the routes driven in the simulation seem to be correct, also their distances written to `plans.xml`.

Regarding the reference system I was using
UTM Zone 32 N (EPSG:25832), which should be okay. I also added a new enum in `UserReferenceFrames` for my implementation. 

I will try your suggestion an do some test with the actual master from this repo.

Thanks so far!

**Comment by JoelGaechter** [comment]
Hey Michael
Apparently there was an unnoticed bug introduced a while ago. This should now be fixed with the new `1.9.1` version of `amodeus`.
There will still be some inaccuracies given the way the sampling works.
Let us know if there are still problems.

**Comment by michaelwittmann** [comment]
@JoelGaechter  thanks for your support. I works now!
